packages/node_modules/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -789,7 +798,70 @@ describe('plugin-meetings', () => {
               meeting: sinon.match.instanceOf(Meeting), type: 'test meeting added type'
             });
           });
+
+          it('creates the meeting from a successful meeting info fetch with random delay', async () => {
```

Comment:
we should also test that we don't do the random delay in these cases:
- when enableUnifiedMeetings is disabled
- when meeting is already active
- when meeting start time is in the past